service-admin/docker/web/Dockerfile

```
@@ -1,12 +1,22 @@
 FROM nginx:stable-alpine
 
+RUN chown -R nginx:nginx /var/cache/nginx && \
+        chown -R nginx:nginx /var/log/nginx && \
+        chown -R nginx:nginx /etc/nginx/conf.d && \
+        touch /run/nginx.pid && \
+        chown -R nginx:nginx /run/nginx.pid
+
+RUN chmod -R 777 /etc/nginx/conf.d
```

Should this be `777` instead of `774 or 770` I think it's a bit too broad otherwise. As we only want users & groups access & not others? If others do need access then it should be RO

---

yep had same comment on the other one - will adjust as this change should read:
```suggestion
RUN chmod o+rwx /etc/nginx/conf.d
```